update/v1alpha1/types_update_status.go

```diff
@@ -0,0 +1,750 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// UpdateStatus is the API about in-progress updates, kept populated by Update Status Controller by
```

Comment:
nit, maybe:

> UpdateStatus reports status for in-progress ClusterVersion updates.

or similar?  Motivation for individual pivots:

* All Kube types are APIs, so "is the API about" doesn't seem to add much context.
* "in-progress updates" didn't explicitly call out ClusterVersion updates (which are all that is in-scope for this API).  Clusters include other updates (workload Deployments, admin-added MachineConfigs triggering MachineConfigPool rolls, etc.), and we want to be clear that those are out of scope.
* "kept populated by..." seems like useful debugging context, but we don't need to commit to a particular backing implementation in this API, we just need enough clarity to enable consumers.  Leaving "how we provide our side of this" uncommitted gives us the flexibility to change the implementation as long as we don't shift the consumer-facing API (although the current _Compatibility level 4_ gives us the ability to shift the consumer-facing API too).

Reply:
I'll go with 

> UpdateStatus reports status for in-progress cluster version updates

I don't want to call out `ClusterVersion` - the initial implementation covers standalone clusters only (so ClusterVersion but I think that's just implementation detail) and the aspiration for the API is to cover updates in general. I added "cluster version updates" to leave other kinds of "updates" out of scope.

Reply:
Works for me; you can mark this thread resolved :)